### ISSUE TITLE
v-v- is parsed and suggested incorrectly

### ISSUE THREAD

**Issue by juliusfriedman** [vuejs/eslint-plugin-vue#2856]
- **ESLint version:** 
- vue-eslint-parser@^10.2.0
- eslint-plugin-vue@^10.3.0
- **Vue version:** 
- 3.5.18
- **Node version:** 
- 22.17.1
- **Operating System:** 
Windows 11

**What did you do?**	

```vue

<div :expand="true" v-v-if="" />

```

**What did you expect to happen?**
[Error unexpected token](https://github.com/vuejs/vue-eslint-parser/blob/4e48327385fd26f39884c81fb21f3f12c36260ad/src/template/index.ts#L166)

**What actually happened?**
Attribute suggested to be moved before `:expand` attribute

[I found while looking into why strict boolean expressions was not working](https://github.com/vuejs/eslint-plugin-vue/issues/2143)

If you need configuration or repo let me know but this can be observed in any configuration /repo using eslint-plugin-vue.

Thanks!

**Comment by waynzh** [comment]
The way `v-v-if` is parsed is expected. However, if you try something like `v-=""`, it will trigger the check you mentioned, resulting in an "unexpected token" error.

**Comment by waynzh** [comment]
I'll close this issue for now since we need more information to debug.
